packages/wrangler/src/dev/miniflare.ts

```diff
@@ -418,6 +418,17 @@ async function buildMiniflareOptions(
 
 		log,
 		verbose: logger.loggerLevel === "debug",
+		handleRuntimeStdio: (stdout, stderr) => {
+			stdout.on("data", (chunk) => {
+				// TODO: with some conditional heuristic, call log.debug instead of log.info
+				log.info(chunk.toString());
+			});
+
+			stderr.on("data", (chunk) => {
+				// TODO: with some conditional heuristic, call log.debug instead of log.error
+				log.error(chunk.toString());
+			});
+		},
```

Comment:
Instead of log.debug like these comments mention, instead we could directly call `appendToDebugLogFile`